### PR TITLE
Gnosis multisig 

### PR DESCRIPTION
**Pull request by Mike-S-Mike**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [ ] Sanity check passed
- [ ] Self-review passed
- [ ] Version updated

WIP

### CODE REVIEW

File: contracts/Front/MultiSig.sol [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+pragma solidity ^0.4.23;
+
+import "multisig-wallet-gnosis/contracts/MultiSigWallet.sol";
+
+import "../Configuration/Config.sol";
+import "../Boilerplate/Head.sol";
+import "../Boilerplate/Multiplexer.sol";
+
+
+contract MultiSig is Base, MultiSigWallet {
```
Comment (Mike-S-Mike) [inline contracts/Front/MultiSig.sol]: Ivan Rukhavets [4:55 PM]
1. The multiplexer should not be a part of context and not a Base class

File: contracts/Front/MultiSig.sol [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        multiplexer = _multiplexer;
+        config = _config;
+
+        for (uint32 i = 0; i < config.C_LEVEL_NUMBER(); i++) {
+            cLevelOwners.push(_owners[i]);
+        }
+    }
+
+    function isCLevelConfirmed(uint transactionId) public view returns (bool) {
```
Comment (Mike-S-Mike) [inline contracts/Front/MultiSig.sol]: Ivan Rukhavets [4:55 PM]
2. As far as we know, we are not doing the C level approvals

File: scripts/postInstall.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+mkdir node_modules/multisig-wallet-gnosis node_modules/multisig-wallet-gnosis/contracts 
+curl https://raw.githubusercontent.com/gnosis/MultiSigWallet/master/contracts/MultiSigWallet.sol -o ./node_modules/multisig-wallet-gnosis/contracts/MultiSigWallet.sol
```
Comment (Mike-S-Mike) [inline scripts/postInstall.sh]: Ivan Rukhavets [4:55 PM]
3. Downloading multisig with curl from github seems like a huge security issue
So what we propose is to use the unchanged Gnosis contract and deploy it separately from other contracts

See this branch https://github.com/ambrosus/ambrosus-node-contracts/tree/multisig_actions

File: contracts/Front/MultiSig.sol (hunk 1 above)
Comment (H-Pavel) [reply contracts/Front/MultiSig.sol]: MultiSigWallet.sol added separatly out of context.

File: contracts/Front/MultiSig.sol (hunk 2 above)
Comment (H-Pavel) [reply contracts/Front/MultiSig.sol]: Removed.

File: scripts/postInstall.sh (hunk 3 above)
Comment (H-Pavel) [reply scripts/postInstall.sh]: MultiSigWallet.sol added separatly.
